artcommon/tests/test_exectools.py

```
@@ -228,6 +228,54 @@ async def test_cmd_assert_async(self):
             create_subprocess_exec.assert_awaited_once_with(*cmd, cwd=fake_cwd)
             self.assertEqual(rc, 0)
 
+    @mock.patch("artcommonlib.exectools.cmd_assert_async")
+    async def test_manifest_tool_uses_explicit_auth_file(self, cmd_assert_async: mock.AsyncMock):
+        await exectools.manifest_tool(
+            ["push", "from-spec", "--", "/tmp/manifest-list.yaml"],
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/manifest-list.yaml"

(S108)